packages/url_launcher/url_launcher_android/android/src/main/java/io/flutter/plugins/urllauncher/UrlLauncher.java

```diff
@@ -118,6 +128,38 @@ public void closeWebView() {
     applicationContext.sendBroadcast(new Intent(WebViewActivity.ACTION_CLOSE));
   }
 
+  private static @NonNull Boolean openCustomTab(
```

Comment:
`@NonNull Boolean` -> `boolean`

The methods above use the other pattern only because it's what Pigeon generates.